apps/supernova/src/App.tsx

```diff
@@ -7,6 +7,7 @@ import React from "react"
 
 import { AppShellProvider, CodeBlock } from "@cloudoperators/juno-ui-components"
 import AppContent from "./AppContent"
+// @ts-expect-error TS(2792) FIXME: Cannot find module './styles.scss?inline'. Did you... Remove this comment to see the full error message
```

Comment:
You need to reference `vite/client` types in `index.d.ts` please check my [PR](https://github.com/cloudoperators/juno/pull/751/files#diff-dbb9f28efe4a65a2992dd02c56c833776e5e8de2d9bf5f39066671b9e42d945eR7) or the `carbon` app.

Reply:
good thing!